components/BriefSection/BriefSection.tsx

```diff
@@ -0,0 +1,70 @@
+import style from "./BriefSection.module.css";
+import Icon from "../Icon/Icon";
+import Title from "../Title/Title";
+
+const BriefSection = () => {
+  const cardArray: any = [
```

Comment:
can you create a sample.json and create this array on there? then you can import it here and i believe code will be cleaner on this component

Reply:
I created a new json to storage data and exported into component